lib/rules/no-invalid-double-slash-comments/__tests__/index.mjs

```diff
@@ -75,6 +89,33 @@ testRule({
 			endLine: 1,
 			endColumn: 13,
 		},
+		{
+			code: stripIndent`
+				.a, // Comment 1
+				/* foo */
+				.b, // Comment 2
+				.c {
+					color: red;
+				}
+			`,
+			description: 'regular comments interleaved with double slash comments',
+			warnings: [
+				{
+					message: messages.rejected,
+					line: 1,
+					column: 5,
+					endLine: 2,
```

Comment:
I am not sure if the `endLine` here is correct or helpful, but out of scope for this issue. This issue is limited to incorrect ranges as a result of interleaved comments. It might be that this is fine :)

Reply:
It has something to do with the trick of using `{}`.
i.e. `// bar {}` vs `// bar`

Reply:
It has something to do with the trick of using `{}`.
The others tests—except l74–83—all use a `{}` construct.
i.e. `// bar {}` vs `// bar`

So we should either have the real commented code (including the following construct) or the part before the `\n` selected.
i.e. either `// Comment 2\n` or `// Comment 2\n.c {\ncolor: red;\n}` (not `// Comment 2\n.`)
Either way it should be fixed.
From an UX standpoint it should be `// Comment 2\n`.

tl;dr `(lines.length > 1)` is false when using `{}`